ocean_lib/ocean/ocean_compute.py

```diff
@@ -216,7 +235,7 @@ def check_output_dict(output_def, consumer_address, data_provider, config: Confi
         default_output_def.update(output_def)
         return default_output_def
 
-    def create_compute_service_descriptor(self, attributes):
+    def create_compute_service_descriptor(self, attributes: Dict) -> ServiceDescriptor:
```

Comment:
Is `attributes: Dict` valid?  I think `Dict[str, Any]` or `dict` would be better.